CONTRIBUTING.md

```diff
@@ -24,10 +24,26 @@ The CSS to be packaged should include css that is necessary to view components i
 - Method that return elements should have a name with Element suffix
 
 ## Release
+
+### Library
 For release, the library configuration of webpack must be used. 
 
 ```
 webpack --config webpack.config.library.js
 ```
 
 Please ensure that any entry level functions are exposed in src/library-entry.js as well as src/entry.js
+
+### Mac OS X Desktop Application
+
+1. Obtain a copy of the [Electron binaries](https://github.com/electron/electron/releases) (e.g. `electron-vx.x.x-darwin-x64.zip`) and decompress
+2. Rename the Electron binary folder and Electron binary to HyPhy Vision and Hyphy Vision.app, respectively
+3. TBD: Get icon working
+4. `cd Hyphy\ Vision/Hyphy\ Vision.app/Contents/Resources/`
+5. `mkdir app && cd app`
+5. `git clone https://github.com/veg/hyphy-vision .`
+6. `yarn`
+7. `webpack`
```

Comment:
May want to include the full path as a reminder just in-case, `./node_modules/.bin/webpack`